### PR TITLE
fix: improving digital job postings

### PR DESCRIPTION
**Pull request by BlueAcidFrog**
## Checklist

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md)
      and signed the CLA.
- [x] I have referenced an issue containing the design document if my change
      introduces a new feature.
- [x] I have read the [security policy](../security/policy).
- [ ] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security vulnerability, I
      confirm that I got green light (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [ ] I have added tests that prove my fix is effective or that my feature
      works.
- [ ] I have added necessary documentation within the code base (if
      appropriate).

### CODE REVIEW

**Comment by BlueAcidFrog** [comment]
First two commits can be ignored - already in PR #490 (8adc6da & deec56a)

**Review by marneysof — APPROVED** [review]

**Review by marneysof — APPROVED** [review]

**Review by marneysof — DISMISSED** [review]

**Review by marneysof — APPROVED** [review]

File: src/markdown/jobs/growth-marketing-manager.mdx [hunk 1]
```diff
@@ -31,41 +31,28 @@ marketing, and data analytics.
...
-- Entrepreneurial mindset and very important, hands-on attitude
-- Detail oriented, a bias for action, and project management skills
-- Enjoy collaboration across departments
+- Experience with modern digital growth strategies and technical/industry experience focused on SEO, SEM, paid social and/or web conversion rate optimization to improve lead generation, sales pipeline, and revenue.
+- Understanding performance data on a deep level (CAC, LTV, Conversion rate, ROAS (return on ad spend) Customer retention rates and churn).
+- Experience in successfully building and improving paid demand generating campaigns for product-focused SaaS enterprises in the B2B market.
+- Strong analytical skills with an ability to compile and analyze campaign data to inform strategic decisions at any part of the funnel.
+- Experience working closely with technical product marketing managers to amplify key content and features.
+- Experience leading cross-department projects and collaborating with web developers to drive meaningful impact.
+- Entrepreneurial mindset, hands-on attitude and very important, detail orientated
```
Comment (vinckr) [inline src/markdown/jobs/growth-marketing-manager.mdx:53]:
```suggestion
- Entrepreneurial mindset, hands-on attitude and very important, detail orientated.
```

File: src/markdown/jobs/growth-marketing-manager.mdx [hunk 2]
```diff
@@ -31,41 +31,28 @@ marketing, and data analytics.
...
+- Fun collaboration across departments
```
Comment (vinckr) [inline src/markdown/jobs/growth-marketing-manager.mdx:54]:
```suggestion
- Fun collaboration across departments.
```

"Fun" 😁

File: src/markdown/jobs/growth-marketing-manager.mdx [hunk 3]
```diff
@@ -31,41 +31,28 @@ marketing, and data analytics.
...
+- Strong editorial skills in English
```
Comment (vinckr) [inline src/markdown/jobs/growth-marketing-manager.mdx:55]:
```suggestion
- Strong editorial skills in English.
```

File: src/markdown/jobs/growth-marketing-manager.mdx [hunk 4]
```diff
@@ -1,9 +1,9 @@
 ---
 published: true
-path: '/jobs/growth-marketing-manager/'
-title: 'Growth Marketing Manager'
+path: '/jobs/digital-growth-manager/'
```
Comment (vinckr) [inline src/markdown/jobs/growth-marketing-manager.mdx:3]:
```suggestion
path: '/jobs/growth-marketing-manager/'
```

As discussed probably easier to leave this as it is.

File: src/markdown/jobs/marketing-campaign-manager.mdx [hunk 5]
```diff
@@ -1,9 +1,9 @@
 ---
 published: true
-path: '/jobs/marketing-campaign-manager/'
-title: 'Marketing Campaign Manager'
+path: '/jobs/digital-campaign-manager/'
```
Comment (vinckr) [inline src/markdown/jobs/marketing-campaign-manager.mdx:3]:
```suggestion
path: '/jobs/marketing-campaign-manager/'
```

**Review by vinckr — COMMENTED** [review]
A few minor changes. 
I will see that the unneeded commits are removed.

**Review by vinckr — APPROVED** [review]
LGTM

File: src/markdown/jobs/growth-marketing-manager.mdx [hunk 6]
```diff
@@ -31,40 +33,45 @@ marketing, and data analytics.
...
-- Build programs and digital campaigns to help people understand, try, and adopt
-  Ory products.
-- Optimize through experimentation. Create A/B tests to iterate on content to
-  increase engagement and product adoption.
-- Distribute Ory Corp core messaging across digital channels. Create and expand
-  digital channels to engage prospects and customers.
+- Build customer focused programs and digital campaigns to help drive
+  data-driven strategic campaign recommendations.
+- Create and expand digital channels to engage prospects and customers to
+  quickly scale acquisition strategy and frive retention.
```
Comment (MicLeey) [inline src/markdown/jobs/growth-marketing-manager.mdx:39]:
```suggestion
  quickly scale acquisition strategy and drive retention.
```

File: src/markdown/jobs/marketing-campaign-manager.mdx [hunk 7]
```diff
@@ -14,48 +14,56 @@ position: 'Full Time'
...
+## Digital Campaign Manager
 
 <About />
 
-As Marketing Campaign Manager you will strategize, execute, and optimize
-data-driven demand generation campaigns on an international level to expand
-Ory’s reach among prospects and customers across multiple channels.
+The Digital Campaign Manager is responsible for developing strategic plans for
+communicating Ory’s brand message on an international level to expand reach
+among prospects and customers across multiple channels. This includes developing
```
Comment (MicLeey) [inline src/markdown/jobs/marketing-campaign-manager.mdx:23]:
```suggestion
among prospects and customers across multiple channels. This includes development
```

**Review by MicLeey — APPROVED** [review]

**Review by vinckr — APPROVED** [review]

**Pull request merged**